integrated_channels/integrated_channel/management/commands/reset_csod_remote_deleted_at.py

```diff
@@ -25,21 +25,20 @@ class Command(IntegratedChannelCommandMixin, BaseCommand):
 
     def handle(self, *args, **options):
         """
-        Mark for re-send all CSOD content transmission with a remote_deleted_at but no api_response_status_code
+        Mark for re-send all CSOD content transmission with a remote_deleted_at
         """
 
         ContentMetadataItemTransmission = apps.get_model(
             'integrated_channel',
             'ContentMetadataItemTransmission'
         )
 
-        csod_deleted_at_but_null_status = Q(
+        csod_deleted_at = Q(
             integrated_channel_code='CSOD',
-            remote_deleted_at__isnull=False,
-            api_response_status_code__isnull=True
```

Comment:
this is set to `200` with the newly refactored code, originally i filtered out stuff that had been "successfully sent" but now, given the timeouts, a bunch of deletes were marked success, given a 200 status, but need to be resent.